extensions/positron-python/src/client/jupyter/jupyterIntegration.ts

```diff
@@ -170,3 +171,108 @@ export class JupyterExtensionIntegration {
         }
     }
 }
+
+export interface JupyterPythonEnvironmentApi {
```

Comment:
Probably worth keeping an eye on what this is solving and whether we can/want to use it.

Reply:
Here's the upstream PR that added it! https://github.com/microsoft/vscode-python/pull/24771

Looks like it was originally added to have one single way to get the environment for Python scripts/notebooks/etc (and fix a bug where the first selection of a kernel for a Jupyter notebook didn't fire a change event).